src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -204,29 +231,26 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
   const showSoundCloudRadio =
     (currentVenue?.showRadio && isSoundCloud) ?? false;
 
+  const showRadio = showNormalRadio || showSoundCloudRadio;
+  const volumeIcon = !volume ? faVolumeMute : faVolumeUp;
+
   return (
     <>
       <header>
         <div className={`navbar navbar_playa ${!isOnPlaya && "nonplaya"}`}>
           <div className="navbar-container">
             <div className="nav-logos">
-              <div className="nav-sparkle-logo">
-                <div />
-              </div>
-              <div
-                className="nav-sparkle-logo_small"
-                onClick={navigateToHomepage}
-              >
-                <div />
-              </div>
               {shouldShowHomeButton && (
                 <FontAwesomeIcon
-                  icon={faHome}
-                  className="NavBar__home-icon"
+                  icon={faCaretLeft}
+                  className="NavBar__home--icon"
```

Comment:
Original class name was OK by BEM standard, the new one signifies `--icon` is a modifier of `__home`

Reply:
fixed 433dfe8